doc/api/crypto.md

```diff
@@ -1902,6 +1902,26 @@ const {
 })();
 ```
 
+### Static method: `KeyObject.isCryptoKey(obj)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `obj` {Object}
+* Returns: {boolean}
+
+Returns `true` if `obj` is a `CryptoKey`, `false` otherwise.
```

Comment:
Wrapping with the `{` `}` brackets allows these to be links to the docs for the object :-)

```suggestion
Returns `true` if `obj` is a {CryptoKey}, `false` otherwise.
```